### ISSUE TITLE
Edge device documentation

### ISSUE THREAD

**Issue by leezu** [apache/incubator-mxnet#18057]
## Description
ARM, Android and Jetson toolchains have been simplified on CI in https://github.com/apache/incubator-mxnet/commit/fb73a1717acad61caeaeef010faed9e9fcc05f0e

Based on the improved setup, we should update
https://mxnet.apache.org/get_started?platform=devices&

In summary, we can recommend using
- For ARM
  - The standard Debian/Ubuntu toolchains on ARMv7 / ARMv8.
  - https://toolchains.bootlin.com/ for ARMv6
- For Android
  - NDK19+ prebuilt toolchain (https://developer.android.com/ndk/guides/other_build_systems)
- For Jetson
  - Jetpack 4.3 + Cuda 10

Each of these is implicitly documented in the Dockerfiles.

cc @ehsanmok in case you are interested in this feature.

**Comment by kmfrick** [comment]
Piggybacking on this: the pre-built library linked [here, section "Install MXNet for Python"](https://mxnet.apache.org/get_started/jetson_setup) is outdated  as JetPack 4.3 bundles CUDA 10.2 and that library is linked against CUDA 10.0 so as soon as you run any script that uses MX you get linker errors. Right now I'm building from source on my Jetson (branch v1.4.x), let me know if there is anything I can do to help.

**Comment by leezu** [comment]
@kmfrick you can consider making a PR to update https://github.com/apache/incubator-mxnet/blob/master/docs/static_site/src/pages/get_started/jetson_setup.md

If you don't need MXNet 1.4 in particular, I'd rather recommend you to build 1.6, or even the per-release 1.7. It's unfortunate that above guide "hardcoded" the 1.4 version when it was written, instead of referring to the latest stable version.

**Comment by kmfrick** [comment]
Yeah I'm building 1.6 now, I noticed half-way through the build that 1.4 was not the most recent release. I'll make a PR against that document when I have some time. Thank you!

**Comment by JustinhoCHN** [comment]
@kmfrick Hi, any update here? I can install mxnet v1.6 on jetson nano with jetpack 4.3, as link here: #17923  but I haven't try it on jetpack 4.4, I prefer building a docker image, though no luck so far.
BTW, cross compile didn't work for me.

**Comment by leezu** [comment]
@mseth10
